examples/Dockerfile.centos7

```diff
@@ -31,8 +20,15 @@ RUN yum -y install \
 # impact. In the interest of time, we demonstrate this by upgrading a single
 # package, though "yum upgrade" seems to pretty reliably tickle the problem.
 # See issue #472.
-RUN          yum -y install fakeroot
-RUN fakeroot yum -y upgrade dbus
-
-# Clean up.
-RUN yum clean all
+ RUN    yum -y install epel-release \
+    && yum -y install \
```

Comment:
These two `yum`s appear to be misaligned.

Additionally, I prefer an additional indentation on the packages. For example:

```
RUN    yum install epel-release \
    && yum install bats \
                   gcc \
                   make \
                   python36 \
                   [...]
 
```

@heasterday @reidpr What are your thoughts?

Reply:
You are correct, I went ahead and corrected the misalignment. Good catch!

As for the additional indentation I can see the appeal. I do worry it would look odd in cases like the following though from `Dockerfile.openmpi`
```dockerfile
RUN    dnf install -y --setopt=install_weak_deps=false \
           automake \
           file \
           gcc \
```
As this would become the following if I understand the new style you are suggesting.
```dockerfile
RUN    dnf install -y --setopt=install_weak_deps=false automake \
                                                                                        file \
                                                                                        gcc \
```

Reply:
I think for hairy cases like that it would be something like:

```
RUN    dnf install -y --setopt=install_weak_deps=false \
                      automake \
                      file \
                      gcc \
```

Reply:
What about we remove the extra whitespace for the `&&` so something like this:
```dockerfile
RUN yum install -y epel-release \
 && yum install -y \
                bats \
                gcc \
                make \
                python36 \
                rpm-build \
                rpmlint \
                wget \
                fakeroot \
 && fakeroot yum -y upgrade dbus \
 && yum clean all
```
@reidpr @j-ogas

Reply:
I would be OK with this, provided we're consistent across the Dockerfiles.